src/WebJobs.Script.WebHost/Models/ExtensionsRestoreJob.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Microsoft.Azure.WebJobs.Script.WebHost.Models
+{
+    public class ExtensionsRestoreJob
```

Comment:
Can you align the plurality of these "ExtensionXXX" types? E.g your enum ExtensionRestoreStatus is singular, this class is plural.